migrator/migrations/3_add_user_for_api.up.sql

```diff
@@ -0,0 +1,2 @@
+CREATE USER gateway
+WITH PASSWORD 'gateway';
```

Comment:
Is it possible to push the password to migration tool and reuse it here instead of hardcode?